src/App.css

```diff
@@ -58,12 +60,15 @@ li {
 }
 
 .singleBook {
+  background: #f5f5f5;
   display: flex;
-  justify-content: space-between;
+  justify-content: space-around;
+  align-items: center;
   border: 1px solid #dcdcdc;
```

Comment:
- [ ] Great job so far 👏🏻 However the Styling must be EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/). kindly improve your text color, center your content header, main and footer, and main page, you also need to improve you margin-top and margin-botton as well to match the original design. Make sure the main page is aligned in the center, you need to fix the margin-left and margin-right and you need to add the missing part eg: the percentage number👍🏻 

- Your Design:

![Screenshot (1)](https://user-images.githubusercontent.com/79058364/209372775-6210d4c8-77ac-424b-a31d-6f6d85a6a01a.png)

- Original Design:

![image](https://user-images.githubusercontent.com/79058364/191857667-6f501420-6d4a-41ab-a615-f0829889a246.png)

Reply:
> * [ ]  Great job so far 👏🏻 However the Styling must be EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/). kindly improve your text color, background color of your `delete book` btns, center your content header, main and footer, and main page, you also need to improve you margin-top and margin-botton as well to match the original design. Make sure the main page is aligned in the center, you need to fix the margin-left and margin-right and you need to add the missing part eg: the percentage number👍🏻
> * Your Design:
> 
> ![Screenshot (1)](https://user-images.githubusercontent.com/79058364/209372775-6210d4c8-77ac-424b-a31d-6f6d85a6a01a.png)
> 
> * Original Design:
> 
> ![image](https://user-images.githubusercontent.com/79058364/191857667-6f501420-6d4a-41ab-a615-f0829889a246.png)

- [ ] Kindly implement the suggestions by the previous reviewer. :+1: